layout-server/src/main/java/org/finos/vuu/layoutserver/repository/MetadataRepository.java

```diff
@@ -6,5 +6,7 @@
 
 import java.util.UUID;
 
+
 @Repository
-public interface MetadataRepository extends CrudRepository<Metadata, UUID> {}
+public interface MetadataRepository extends CrudRepository<Metadata, UUID> {
+}
```

Comment:
I'd pop this back. But I'm pedantic.

```suggestion
public interface MetadataRepository extends CrudRepository<Metadata, UUID> {}
```